samples/musicfestival-frontend-react/src/components/SearchButton.tsx

```diff
@@ -31,7 +30,7 @@ function SearchButton(): JSX.Element {
 
     function onValueChange(event: any){
         setSearchValue(event.target.value);
-        event.target.value != "" && event.target.value !== undefined ? setIsShown(true) : setIsShown(false);
+        event.target.value !== undefined ? setIsShown(true) : setIsShown(false);
```

Comment:
with Boolean methods, usually you do `isShown` or even `shown`. No need to add a set prefix.

Reply:
This is how hooks are defined in react. Hooks are a new addition to react 16.8
Normally, hooks are defined like this: 
`const [something, setSomething] = useState()`
`something` is a state variable, and `setSomething` is the function we will use to update the state.
Ofcourse, we can change `setSomething` to any text that we want, but I'd like to keep it this way, for naming convention related to React.

More info: https://www.w3schools.com/react/react_usestate.asp